tests/unit/v1/test_collection.py

```diff
@@ -218,7 +224,7 @@ def _next_page(self):
         collection = self._make_one("collection", client=client)
 
         if page_size is not None:
-            documents = list(collection.list_documents(page_size=page_size))
+            documents = list(collection.list_documents(page_size))
```

Comment:
Is `page_size` a positional-only parameter now? If so, I'm curious why.

Reply:
it shouldn't be. Let me undo this one line.